### DISCUSSION TITLE
yml file for Docker

### DISCUSSION THREAD

**Discussion by Anriell** [a13xh7/QaraTMS#13] [Q&A] [answered]
Hello! Could you please share yml file for QaraTMS?

**Comment by a13xh7** [comment]
Hello,
I didn't use docker so there is no yml file for it.

**Reply by TkachenkoVitaliy** [reply]
Привет я сделал форк твоего репозитория для создания docker-compose.yml (https://github.com/TkachenkoVitaliy/QaraTMS). Но есть одна проблема, можешь пожалуйста подсказать что нужно изменить в коде чтобы исполнение php artisan db:seed --class=AdminSeeder было только если таблица пользователей пустая. Так иначе нужно либо вручную один раз использовать его через exec docker контейнера. Либо оно будит крашиться при перезапуске контейнера изза того что уж есть пользователь с такими данными.

**Reply by a13xh7** [reply]
Привет, я почти не работал с докером, не могу подсказать.

**Comment by a13xh7** [comment, marked as answer]
Thanks to @TkachenkoVitaliy there are docker files now
